O/openPMD_api/build_tarballs.jl

```diff
@@ -73,8 +73,8 @@ cmake \
     -DBUILD_CLI_TOOLS=OFF \
     -DBUILD_EXAMPLES=OFF \
     -DBUILD_TESTING=OFF \
-    -DJulia_PREFIX=${prefix} \
-    -DopenPMD_USE_Julia=ON \
+    -DJULIA_PREFIX=${prefix} \
```

Comment:
The `Julia_PREFIX` was the correct case here.

Reply:
Yes it was. Thanks for spotting this!